<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="0" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="datEnd">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCommandLinkButton" name="btnDeparture">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Départ</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QCommandLinkButton" name="btnReturn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Date début</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Date Fin</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="4">
     <widget class="QTextEdit" name="apiLog">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="4" colspan="4">
     <widget class="QTableView" name="tblEvent">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDateEdit" name="datStart">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="txtSearch"/>
    </item>
    <item row="1" column="2" rowspan="2" colspan="2">
     <widget class="QCommandLinkButton" name="commandLinkButton_3">
      <property name="text">
       <string>Centrer sur aujourd'hui</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionAfficher_log_REST"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAfficher_log_REST">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Afficher log REST</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>datStart</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>MainWindow</receiver>
   <slot>startDateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datEnd</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>MainWindow</receiver>
   <slot>endDateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>centerOnToday()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReturn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startDeparture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblEvent</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectionChanged(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAfficher_log_REST</sender>
   <signal>toggled(bool)</signal>
   <receiver>apiLog</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startDateChanged()</slot>
  <slot>endDateChanged()</slot>
  <slot>centerOnToday()</slot>
  <slot>startDeparture()</slot>
  <slot>selectionChanged(QModelIndex)</slot>
 </slots>
</ui>
